checkstyle/import-control.xml

```diff
@@ -144,15 +155,14 @@
     </subpackage>
 
     <subpackage name="record">
-      <allow pkg="net.jpountz" />
+      <allow pkg="org.apache.kafka.common.compression" />
```

Comment:
Could we move `CompressionType` to `org.apache.kafka.common.compression` and then make both `ZstdFactory` and SnappyFactory be package-private?

Reply:
Possibly. Let me see what other impact it has.

Reply:
This affects a lot of files and I'd rather do that in a separate PR as I may backport this one to stable branches.

Reply:
make sense.